### ISSUE TITLE
Dynamic graph doc page is outdated

### ISSUE THREAD

**Issue by scholer** [gephi/gephi#1336]
The documentation page for dynamic graphs at https://gephi.org/gexf/format/dynamics.html is out of date: Neither of the two referenced gefx files work with Gephi 0.90. 
Opening the first file gives three critical errors, and the timeline is not available ("Graph is not dynamic").
The second graph uses open-ended time intervals which are no longer supported.

**Comment by duncdrum** [comment]
@scholer If you would like to have a look or add the documentation is moving to the wiki in this repo https://github.com/gephi/gexf

**Comment by eduramiba** [comment]
This should be fixed with #1337. Warnings for open intervals will still appear, but the import will be done correctly.

The graph is not dynamic problem is another problem and we need to check it.

**Comment by duncdrum** [comment]
@edurimba I saw your fix. Just wanted to point scholer to the wiki, since collaborating on the documentation is the whole point of the wiki is it not?

**Comment by eduramiba** [comment]
Yeah, I was just commenting that the new version is supposed to be compatible with old gexf but wasn't due to a bug. We also need to update documentation so we keep this issue open.
